scripts/releaser/release-debian.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# This script is interactive as you need to log in to 'az'.
+
+from __future__ import print_function
+
+import os
+import sys
+import time
+from datetime import datetime
+from six import StringIO
+from sh import az, ssh
+
+
+script_env = {}
+
+def add_script_env(name):
+    script_env[name] = os.environ.get(name)
+
+add_script_env('REPO_NAME')
+add_script_env('CLI_VERSION')
+add_script_env('CLI_DOWNLOAD_SHA256')
+add_script_env('AZURE_STORAGE_CONNECTION_STRING')
+add_script_env('DEBIAN_REPO_ID')
+add_script_env('DEBIAN_REPO_URL')
+add_script_env('DEBIAN_REPO_USERNAME')
+add_script_env('DEBIAN_REPO_PASSWORD')
+
+assert (all(script_env[n] != None for n in script_env)), "Not all required environment variables have been set.  {}".format(script_env)
```

Comment:
`"{}".format(script_env)` would print the dict so key/value pair for each env var so you'd see which ones are not set.